### PR TITLE
Update Discosnp to v2.3.0

### PR DESCRIPTION
**Pull request by cmonjeau**

### CODE REVIEW

File: tools/colibread/discosnp_pp.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version='1.0' encoding='utf-8'?>
-<tool profile="16.04" id="discosnp_pp" name="DiscoSnp++" version="2.2.10">
+<tool profile="16.04" id="discosnp_pp" name="DiscoSnp++" version="2.3.0">
     <description>is an efficient tool for detecting SNPs without a reference genome.</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <requirements>
-        <requirement type="package" version="2.2.10">discosnp</requirement>
+        <requirement type="package" version="2.3.0">discosnp</requirement>
```
Comment (bgruening) [inline tools/colibread/discosnp_pp.xml]: Do you want to put the version number also in a macro? You can use it here and in the tool version.
Also the entire requirement could move into the macro as well as citation and parts of the help?

File: tools/colibread/discosnp_RAD.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            @discosnp_paired_reads@
+        #end if
+
+        ## get the absolute path for short_reads_connector
+        CONNECTOR_PATH=`which short_read_connector.sh` &&
+        CONNECTOR_DIR=`dirname \$CONNECTOR_PATH` &&
+
+        run_discoSnpRad.sh
+        @discosnp_inputs@
+        -S \$CONNECTOR_DIR
```
Comment (bgruening) [inline tools/colibread/discosnp_RAD.xml]: Is this folder just used for reading or also for writing?

Comment (bgruening) [reply tools/colibread/discosnp_RAD.xml]: I guess it needs to be double quoted in cases where the path contains white spaces.

Comment (cmonjeau) [reply tools/colibread/discosnp_RAD.xml]: The folder is just for reading. I added the double quoted.

File: tools/colibread/discosnp_pp.xml (hunk 1 above)
Comment (cmonjeau) [reply tools/colibread/discosnp_pp.xml]: Done

**Comment by bgruening** [comment]
Nice thanks a bunch @cmonjeau!

**Comment by bgruening** [comment]
@cmonjeau for some reason the tests failed on master. Can you look at it? https://travis-ci.org/galaxyproject/tools-iuc/jobs/306399323

**Comment by nsoranzo** [comment]
@cmonjeau To clarify, that means that the `colibread` repositories were not updated on the ToolShed.

**Pull request merged**